server/test/Analysis.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Place to run experiment analysis. Should only be name with .test extension outside of master.
+ */
+
+import { GaWorker } from "../src/ga/gaworker";
+import { Note } from "../src/models/Note";
+
+/**
+ * Evaluates percentage of notes exceeding desired range
+ */
+test("Test notes outside of vocal range", () => {
```

Comment:
I will make this an experiment when you merge in your PR with test framework.

Reply:
Merged, you can go ahead and do that